doc/release/upcoming_changes/20500.new_feature.rst

```diff
@@ -0,0 +1,3 @@
+New parameter `ndmin` added to `genfromtxt`
+-------------------------------------------------------------------------
+This parameter behaves the same as `ndmin` from `loadtxt`.
```

Comment:
```suggestion
New parameter ``ndmin`` added to ``genfromtxt``
-------------------------------------------------------------------------
This parameter behaves the same as ``ndmin`` from `~numpy.loadtxt`.
```

Reply:
Making appropriate use of single and double quotes should help with intersphinx links.